apps/web/utils/captcha-verify.ts

```diff
@@ -16,7 +16,7 @@ interface RecaptchaResponse {
   'error-codes'?: Errors[] // optional
 }
 
-export default async function captchaVerify(
```

Comment:
q: ive seen this kind of changing defaults to named exports a few times. Is there a benefit beyond aesthetics?